src/base/os.lua

```diff
@@ -836,6 +836,7 @@
 		["solaris"]  = { "solaris", "posix", "desktop" },
 		["uwp"]      = { "uwp", "windows", "desktop" },
 		["windows"]  = { "windows", "win32", "desktop" },
+		[premake.ANDROID]  = { "android", "mobile" },
```

Comment:
Alphabetical order, also none of the others use the premake variables, so this probably shouldn't either for consistency. I think this was overlooked for "posix", perhaps it should have it too.

Reply:
Sorted